performance-evaluation/google_sheets.py

```diff
@@ -4,113 +4,129 @@
 import readchar
 
 
-# Checks the last updated timestamp por template. In case it was modified, script code may have to be adapted
-def validate_updated_timestamp(spreadsheet, timestamp):
-	if spreadsheet.updated != timestamp:
-		print('Fecha de última modificación: ' + spreadsheet.updated)
-		print('Cancelando script. Archivo \'' + spreadsheet.title + '\' modificado, revisar script.')
-		print('')
-		exit()
+def create_sheet(google_credentials, title, folder_key):
+    print('Creando nuevo sheet...')
+    sheet = google_credentials.create(
+        title + ' - ', template=None, folder=folder_key)
+    print('Sheet ' + title + ' creada satisfactoriamente.\n')
+    return sheet
 
 # Authenticate using OAuth from a given credentials file path and return credentials
+
+
 def get_google_credentials():
-	print('Autenticando...')
-	google_credentials = pygsheets.authorize(CREDENTIALS_FILE_PATH)
-	print('Autenticación exitosa!')
-	print('')
-	return google_credentials
+    print('Autenticando...')
+    google_credentials = pygsheets.authorize(GSHEETS_FILE_NAME)
+    print('Autenticación exitosa!\n')
+    return google_credentials
 
 # Open and return the template sheets
+
```

Comment:
Why is there always two emtpy lins between the comment for the function and the function itself? It makes it hard to find what the comment is about and most likely if the functions is moved the comment won't be moved with it. 

If you don't like the comments, you can find some way for your editor to hide them or just remove them entirely

Reply:
This is the linter that i've installed in VSCode. I assumed that it worked like this. But I can changed it.

Reply:
Done